spanner/value.go

```diff
@@ -3209,6 +3209,11 @@ func (w withLenient) Apply(s *decodeSetting) {
 	s.Lenient = w.lenient
 }
 
+// WithLenient returns a DecodeOptions that allows decoding into a struct with missing fields in database.
+func WithLenient() DecodeOptions {
```

Comment:
Why was this not already exported? Was this something that users could not use? Or did they have some other API that they could use to access this option?

Reply:
Earlier there were 2 public methods ToStruct and ToStructLenient, we don't want to carry the approach of having 2 separate methods for SelectAll hence added as option.